src/language-handlebars/printer-glimmer.js

```diff
@@ -156,6 +158,18 @@ function print(path, options, print) {
       return [node.key, "=", print("value")];
 
     case "TextNode": {
+      // Don't format content:
+      // 1.in <pre>,
+      // 2.in <style> tags when there are CSS syntax errors
+      // 3.when `embeddedLanguageFormatting` is disabled
+      if (
+        path.parent.tag === "pre" ||
+        path.parent.tag === "style" ||
+        options.embeddedLanguageFormatting === "off"
+      ) {
+        return node.chars;
+      }
```

Comment:
This is not necessary, if `embeddedLanguageFormatting: 'off'`, we should use the old logic.

Reply:
Fixed in https://github.com/prettier/prettier/pull/15087/commits/6804b38b0c4d271758b95d97284e9615de37fe2d